dig.go

```diff
@@ -241,7 +246,7 @@ func (c *Container) get(e edge) (reflect.Value, error) {
 	}
 
 	if err := c.contains(n.deps); err != nil {
-		return _noValue, fmt.Errorf("missing dependencies for %v: %v", e.key, err)
+		return _noValue, errMissing(fmt.Sprintf("missing dependencies for %v: %v", e.key, err))
```

Comment:
Optional: IMO it would be preferable for `errMissing` to hold a list of missing dependencies and construct the message in `Error()`.